README.md

```diff
@@ -14,16 +14,16 @@
 
 ## Description
 
-The app follows the structure of the four passes of Gibson's methodology. It has been designed with prompts to help the enumerator carrying out the survey to make the process as intuitive as possible. Having been designed with offline usage in mind, it uses a local database of the recipes and recent data collected.
+The app follows the structure of the four passes of Gibson's methodology. It has been designed with prompts to help the enumerator carrying out the survey to make the process as intuitive as possible. Having been designed with offline usage in mind, it stores the recipes and all collected data locally on the device, and allows to export it later.
```

Comment:
```suggestion
The app follows the structure of the four passes of Gibson's methodology. It has been designed with prompts to help the enumerator carrying out the survey to make the process as intuitive as possible. Having been designed with offline usage in mind, it stores the recipes and all collected data locally on the device, allowing it to be exported at a future point.
```

Reply:
Great, I think that since you approve, I can rewrite the docs still in this PR, and then, once it is merged, I'll delete the current `main` branch and rename the current `dev` branch to `main`. Does that sound alright?